### PR TITLE
Render producers list for a group

### PR DESCRIPTION
**Pull request by enricostano**
We take in account the following attributes:
- `can_edit`: whether the `producer` has been created from the current group, for now we show `editar`
- `isProvider`: whether the `producer` can receive orders from the current group, for now we show `pedir`
## Dependency

https://github.com/coopdevs/katuma/pull/92

2016-10-12-151926_1676x1007_scrot

### CODE REVIEW

File: src/containers/Group/Producers/Base.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    const suppliers = suppliersReducer.suppliers.byProducerId;
+
+    return {
+      producers: providers(producers, suppliers),
+    };
+  };
+};
+
+const asyncConnectProps = [{
+  deferred: true,
```
Comment (andresgutgon) [inline src/containers/Group/Producers/Base.js]: This will make all be rendered only on front end

Comment (enricostano) [reply src/containers/Group/Producers/Base.js]: Ok, should I delete the `deferred` key? It's `false` by default I guess.

Comment (andresgutgon) [reply src/containers/Group/Producers/Base.js]: Yes, delete it if you want universal fetch of that info

Comment (enricostano) [reply src/containers/Group/Producers/Base.js]: done...

**Comment by andresgutgon** [comment]
I need this code. I'll cherry pick it here: https://github.com/coopdevs/katuma-web/pull/18

**Pull request closed without merging**